jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/Queue.kt

```diff
@@ -0,0 +1,11 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.sqs
+
+class Queue(queueUrl: String) {
+    val regionId = queueUrl.substringAfter("sqs.").substringBefore('.')
+    val accountId = queueUrl.substringAfter("amazonaws.com/").substringBefore('/')
```

Comment:
China is `sqs.<region>.amazonaws.com.cn`: https://docs.amazonaws.cn/en_us/general/latest/gr/cnnorth_region.html
and govcloud is also different, so we can't rely on amazonaws.com. This doc has the format: https://docs.amazonaws.cn/sdk-for-net/v3/developer-guide/how-to/sqs/QueueURL.html